src/main/java/de/christophkraemer/rhino/javascript/RhinoScriptEngineFactory.java

```diff
@@ -133,7 +138,7 @@ public String getProgram(String... statements) {
         names = new ArrayList<String>(6);
         names.add("js");
         names.add("rhino");
-        names.add("rhino17R5");
```

Comment:
I wouldn't remove this. You can add your name as an additional name. But if you have two different script engines with the same name, this implementation might not be choosen. We need a unique identifier for this script engine.